code/modules/spells/spell_types/mime.dm

```diff
@@ -59,4 +59,83 @@
 		if(H.mind.miming)
 			H << "<span class='notice'>You make a vow of silence.</span>"
 		else
-			H << "<span class='notice'>You break your vow of silence.</span>"
\ No newline at end of file
+			H << "<span class='notice'>You break your vow of silence.</span>"
+
+// These spells can only be gotten from the "Guide for Advanced Mimery series" for Mime Traitors.
+
+/obj/effect/proc_holder/spell/aoe_turf/conjure/mime_wall/blockade
+	name = "Invisible Blockade"
+	desc = "With more polished skills, a powerful mime can create a invisble blockade, blocking off a 3x1 area."
+	summon_type = list(/obj/effect/forcefield/mime/advanced)
+	invocation_type = "emote"
+	invocation_emote_self = "<span class='notice'>You form a blockade around yourself.</span>"
+	summon_lifespan = 600
+	charge_max = 600
+	summon_amt = 0
+
+/obj/effect/proc_holder/spell/aoe_turf/conjure/mime_wall/blockade/Click()
+	if(usr && usr.mind)
+		if(!usr.mind.miming)
+			usr << "<span class='notice'>You must dedicate yourself to silence first.</span>"
+			return
+		var/newturf
+		if(usr.dir == NORTH || usr.dir == SOUTH)

```

Comment:
dude wat, just do for

    (var/turf/T in orange(1))
        spawn_the_forcefield on T)